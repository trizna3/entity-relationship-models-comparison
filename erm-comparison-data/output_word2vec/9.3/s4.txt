students-student
payments-payments
rooms-room
account-accounts
contracts-EMPTY
buildings-hostel;building;floor
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-payments-accounts
MOVE_ATTR_TO_INCIDENT_ENTITY_SET:S-end-room
CHANGE_CARDINALITY:S-building-hostel
CONTRACT_11_ASSOCIATION:S-hostel-building
CHANGE_CARDINALITY:S-floor-building
CONTRACT_11_ASSOCIATION:S-building-floor
