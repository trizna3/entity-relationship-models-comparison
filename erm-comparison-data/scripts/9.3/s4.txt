entity set hostel {
};

entity set building {
	"address"
};

entity set floor {
};

entity set room {
	"capacity"
	"fee"
};

entity set student {
};

entity set accounts {
	"payments"
	"time"
	"credit"
};

association between 
{
	hostel ONE
	building MANY
}{};

association between 
{
	building ONE 
	floor MANY
}{};

association between 
{
	floor ONE 
	room MANY
}{};

association "contract" between 
{
	room ONE 
	student ONE
}{
	"start"
	"end"
};

association between 
{
	student ONE 
	accounts ONE
}{};