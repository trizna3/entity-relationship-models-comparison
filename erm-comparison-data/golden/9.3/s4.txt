buildings-building
rooms-room
contracts-
payments-EMPTY
account-accounts
students-student